api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -21,7 +21,67 @@ public interface Privilege {
 
   /** The name of this privilege. */
   enum Name {
-    /** The privilege of load a catalog. */
-    LOAD_CATALOG
+    /** The privilege to show catalogs. */
+    SHOW_CATALOG,
+    /** The privilege to use a catalog. */
+    USE_CATALOG,
+    /** The privilege to create a catalog. */
+    CREATE_CATALOG,
+    /** The privilege to alter a catalog. */
+    ALTER_CATALOG,
+    /** The privilege to drop a catalog. */
+    DROP_CATALOG,
```

Comment:
Maybe we can also add a bit to each privilege like `0x01`, `0x10`, so that later on when we have a privilege like `ALL_PRIVILEGE` that can be represented as a combination of several privileges, you can think of this.